commands/dev-deploy.js

```diff
@@ -0,0 +1,66 @@
+const { KeyPair } = require('nearlib');
+const exitOnError = require('../utils/exit-on-error');
+const connect = require('../utils/connect');
+const { readFile, writeFile } = require('fs').promises;
+
+module.exports = {
+    command: 'dev-deploy <wasmFile>',
+    desc: 'deploy your smart contract using temporary account (TestNet only)',
+    builder: (yargs) => yargs
+        .option('wasmFile',{
+            desc: 'Path to wasm file to deploy',
+            type: 'string',
+            default: './out/main.wasm'
+        })
+        .option('init', {
+            desc: 'Create new account for deploy (even if there is one already available)',
+            type: 'boolean',
+            default: false
+        })
+        .alias({
+            'init': ['force', 'f'],
+        }),
+    handler: exitOnError(devDeploy)
+};
+
+async function devDeploy(options) {
+    const { nodeUrl, helperUrl, masterAccount, wasmFile } = options;
+
+    if (!helperUrl && !masterAccount) {
+        throw new Error('Cannot create account as netiher helperUrl nor masterAccount is specified in config for current NODE_ENV (see src/config.js)');
+    }
+
+    const near = await connect(options);
+    const accountId = await createDevAccountIfNeeded({ ...options, near });
+    console.log(
+        `Starting deployment. Account id: ${accountId}, node: ${nodeUrl}, helper: ${helperUrl}, file: ${wasmFile}`);
+    const contractData = [await readFile(wasmFile)];
+    const account = await near.account(accountId);
+    await account.deployContract(contractData);
+    console.log(`Done deploying to ${accountId}`);
+}
+
+async function createDevAccountIfNeeded({ near, keyStore, networkId, init }) {
+    const accountFilePath = `${keyStore.keyDir}/dev-account`;
```

Comment:
Long term, we should still shoot for global file path. This implementation doesn't exclude that.

Reply:
We need to think more on this. I feel that stuff like current dev account likely belongs to project folder (as is project specific), but production keys should better go to some more secure place (e.g. OS X key chain).